README.md

```diff
@@ -1,9 +1,1444 @@
+    operations. For these data manipulation operations, we can specify a condition expression to
+    determine which items should be modified. If the condition expression evaluates to true, the
+    operation succeeds; otherwise, the operation fails.
+
+-   #### Update Expressions
+
+    To update an existing item in an AWS DynamoDB table, we use the Update Item operation. We must
+    provide the key of the item that we want to update. We must also provide an update expression,
+    indicating the attributes that you want to modify and the values that you want to assign to
+    them. There are 4 operations update expression supports:
+
+    -   [SET](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.UpdateExpressions.html#Expressions.UpdateExpressions.SET)
+        —modifying or adding item attributes
+    -   [REMOVE](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.UpdateExpressions.html#Expressions.UpdateExpressions.REMOVE)
+        —deleting attributes from an item
+    -   [ADD](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.UpdateExpressions.html#Expressions.UpdateExpressions.ADD)
+        —updating numbers and sets
+    -   [DELETE](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.UpdateExpressions.html#Expressions.UpdateExpressions.DELETE)
+        —removing elements from a set
+
+-   #### Key Condition Expressions
+
+    They are used when querying a table with a composite primary key to limit the items selected.
+
+-   #### Filter Expressions
+
+    They allow you to filter the results of queries and scans to allow for more efficient responses.
+
+-   #### Projection Expressions
+
+    They are used to specify a subset of attributes you want to receive when reading Items. We used
+    these in our GetItem calls in the previous lesson.
+
+
+
+  
+
+
+
+The following actions uses expressions
+
+
+
+<table>
+<colgroup>
+<col style="width: 50%" />
+<col style="width: 50%" />
+</colgroup>
+<thead>
+<tr class="header">
+<th>Expression Name</th>
+<th>Action Name</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>Condition Expression</td>
+<td><ul>
+<li>Delete Item</li>
+<li>Put Item</li>
+</ul></td>
+</tr>
+<tr class="even">
+<td>Update Expression</td>
+<td><ul>
+<li>Update Item</li>
+</ul></td>
+</tr>
+<tr class="odd">
+<td>Key Condition Expression</td>
+<td><ul>
+<li>Query Data</li>
+</ul></td>
+</tr>
+<tr class="even">
+<td>Filter Expression</td>
+<td><ul>
+<li>Query Data</li>
+</ul></td>
+</tr>
+<tr class="odd">
+<td>Projection Expression</td>
+<td><ul>
+<li>Query Data</li>
+</ul></td>
+</tr>
+</tbody>
+</table>
+
+#### Expression attribute names and values
+
+While using this app certain actions will ask for **expression attribute names** and **expression
+attribute values** which would be necessary while using expressions. This values are necessary to
```

Comment:
```suggestion
attribute values** which would be necessary while using expressions. These values are necessary to
```

Reply:
Updated the code and resolved the issue.